packages/react-scripts/template/src/index.js

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import './index.css';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { Router, browserHistory } from 'react-router';
+import { syncHistoryWithStore } from 'react-router-redux';
+
+import routes from './store/routes';
+import configureStore from './store/configure-store';
+import './styles/index.css';
+
+const store = configureStore({});
+const history = syncHistoryWithStore(browserHistory, store);
+
 
 ReactDOM.render(
-  <App />,
+  <div>
```

Comment:
Agreed. I was honestly just messing with the configs, not looking at the current state of our starter.
Will remove this